package.json

```diff
@@ -27,33 +27,45 @@
     "Google Cloud Memorystore for Redis API"
   ],
   "scripts": {
-    "test": "mocha",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "lint": "eslint '**/*.js'",
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
```

Comment:
to give us a full picture of coverage, I think we should gravitate towards using `c8` for running `system-test` and `sample-tests`, codecov merges reports.

Reply:
OK! updated `package.json`